src/Razor/test/Microsoft.VisualStudio.LanguageServer.ContainedLanguage.Test/TestVirtualDocumentFactory.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Microsoft.VisualStudio.LanguageServer.ContainedLanguage.Test;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServer.ContainedLanguage
+{
+    internal class TestVirtualDocumentFactory : VirtualDocumentFactoryBase
```

Comment:
Can make this a private class in `VirtualDocumentFactoryBaseTest.cs`

Reply:
Sure

---
In reply to: [452551276](https://github.com/dotnet/aspnetcore-tooling/pull/2201#discussion_r452551276) [](ancestors = 452551276)